### PR TITLE
Feature/#959/batch withdrawal frontend

### PR DESCRIPTION
**Pull request by yi-cro**
Fixes https://github.com/crypto-com/chain-desktop-wallet/issues/959

- [x] A CTA button Withdraw All Rewards
- [x] A confirmation modal involving cumulative rewards info

### CODE REVIEW

**Comment by crypto-matto** [comment]
## Issue
- [ ] The `Withdraw All Rewards` modal is not displaying data correctly on first attempt, until open it twice. 
- [ ] Doubled `(s)(s)` in `Withdraw Reward(s) From Validators(s)(s)`. 
- [ ] The withdrawal validator list ordering, together with `Rewards` amount is changed after the `Confirm` button is clicked. 
## Suggestion
- [ ] `Withdraw All Rewards` button could remove the border, or use `type="primary"` instead, since the current button style is intended not to suggests users clicking on it (e.g. `Clear Storage` button in `Settings`) 
- [ ] You could definitely make good use of the existing `REWARD BALANCE` at the top to display total rewards

**Comment by crypto-matto** [comment]
Like this design on validator list btw 👍🏻
reward

**Comment by cdc-Hitesh** [comment]
Awesome work.
Got some feedback.

- [ ]  Had to open withdraw all rewards two times to see the list
- [ ] The index count with Validators list can be removed to avoid confusion
- [ ] `Rewards` can be replaced with `Total rewards`

**Comment by crypto-matto** [comment]
## Issue
- [ ] Hide Withdraw All Rewards button when reward list is empty
hide
hide2

File: src/pages/staking/staking.tsx [hunk 1]
```diff
@@ -1788,6 +1925,31 @@ const StakingPage = () => {
                 : ''}
             </div>
           </div>
+
+          <ModalPopup
```
Comment (crypto-matto) [inline src/pages/staking/staking.tsx:1929]: I'm seeing 2 `unbonding-modal`. Is this one duplicated and added without reason?

**Review by cdc-Hitesh — APPROVED** [review]

**Pull request merged**